units/INC9002/INC9002_unit.bp

```diff
@@ -0,0 +1,155 @@
+UnitBlueprint {
+    Audio = {
+        DeathExplosion = Sound {
+            Bank = 'XSL_Destroy',
+            Cue = 'XSB_Destroy_Lrg_PreDestroy',
+            LodCutoff = 'UnitMove_LodCutoff',
+        },
+        Destroyed = Sound {
+            Bank = 'XSL_Destroy',
+            Cue = 'XSB_Destroy_Huge',
+            LodCutoff = 'UnitMove_LodCutoff',
+        },
+        UISelection = Sound {
+            Bank = 'SeraphimSelect',
+            Cue = 'Seraphim_Select_Structure',
+            LodCutoff = 'UnitMove_LodCutoff',
+        },
+    },
+    Categories = {
+        'NOMAD',
+        'STRUCTURE',
+        'CIVILIAN',
+        'LAND',
+        'VISIBLETORECON',
+        'RECLAIMABLE',
+    },
+    CollisionOffsetX = 0,
+    CollisionOffsetY = 0,
+    CollisionOffsetZ = 0,
+    Defense = {
+        ArmorType = 'Normal',
+        Health = 5000,
+        MaxHealth = 5000,
+        RegenRate = 0,
+    },
+    Description = '<LOC inc9002_desc>Special Crystal',
+    Display = {
+        Mesh = {
+            IconFadeInZoom = 130,
+            LODs = {
+                {
+                    AlbedoName = 'INC9002_Albedo.dds',
+                    LODCutoff = 120,
+                    LookupName = '/textures/environment/Falloff_seraphim_lookup.dds',
+                    ShaderName = 'Unit',
+                },
+            },
+        },
+        PlaceholderMeshName = 'INC9002',
+        SpawnRandomRotation = true,
+        UniformScale = 0.2,
+    },
+    Economy = {
+        BuildCostEnergy = 50,
+        BuildCostMass = 5,
+        BuildTime = 50,
+    },
+    Footprint = {
+        SizeX = 6,
+        SizeZ = 6,
+    },
+    General = {
+        Category = 'Defense',
+        Classification = 'RULEUC_MilitaryVehicle',
+        CommandCaps = {
+            RULEUCC_Attack = true,
+            RULEUCC_CallTransport = false,
+            RULEUCC_Capture = false,
+            RULEUCC_Guard = false,
+            RULEUCC_Move = true,
+            RULEUCC_Nuke = false,
+            RULEUCC_Patrol = false,
+            RULEUCC_Reclaim = false,
+            RULEUCC_Repair = false,
+            RULEUCC_RetaliateToggle = false,
+            RULEUCC_Stop = false,
+            RULEUCC_Transport = false,
+        },
+        FactionName = 'Seraphim',
+        SelectionPriority = 5,
+        TechLevel = 'RULEUTL_Basic',
+        UnitWeight = 1,
+    },
+    Intel = {
+        JamRadius = {
```

Comment:
I'm not quite sure what the point of this is. I just read that it has multiple abilities as well as unlocking abilities (Questionable). The only thing these cystals have done is jam radar.

Original Crystal Name: "Blue Sky Crystal"
Faction needs changing from Seraphim to Nomads

I think this idea needs to be reviewed before it's pushed to our repo.